tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+
+### 2) Enable `Google CloudFunction API`
+Run the following command:
+```
+$ gcloud --project=<YOUR_PROJECT_ID> beta service-management enable cloudfunctions.googleapis.com
+```
+This command might take a few minutes. After it completes the `Google CloudFunctions` API will be enabled for your project.
+
+### 3) Writing the code
+To deploy into `Google CloudFunctions` we will use two files:
+- A `package.json` file that describes the module, dependencies etc'
+- An `index.js` file that exports the code to be run.
+
+### 4) Create a `package.json` file
+Create a file named `package.json`, copy the content from [this link](./package.json) into the file and save it locally.
+
+### 5) Create an `index.js` file
+Create a file named `index.js`, copy the content from [this link](./index.js) into the file and save it locally.
+
+### 6) Deploy the code
+Run the following command:
+```
+$ gcloud --project=<YOUR_PROJECT_ID> beta functions deploy gcp-alert-service --stage-bucket <YOUR_PROJECT_ID>-gcp-alert-service --trigger-topic gcp-alert-service --entry-point=pubsubLogSink --region=us-central1
+```
+This command will take a few minutes. 
+When complete you will can see the deployed function [here](https://console.cloud.google.com/functions/list).
+
+## Configuration
+Altough the code is deployed and events are being pushed - you will not yet recieve any `Slack` notifications. For that to happen the application must be configured.
+
+The application code deployed in the previous section uses [Google Cloud Datastore](https://cloud.google.com/datastore) to store configuration.
+`Google Cloud Datastore` is a fully managed, auto-scaling, serverless no-sql database. It allows fast updates and fetches as well as some degree of query support.
+
+The application needs to have a `Slack API Token` configuration to allow it positn new messages to a channel.
+
+You will also need to define the alerts you wish to be notified of.
+
+### 1) Configuration console
+Open the [Google Cloud Datastore Console](https://console.cloud.google.com/datastore) in your favorite browser.
+
+### 2) Generate a `Slack API Token`
+- Go to [https://<YOUR_SLACK_TEAM>.slack.com/apps/new/A0F7YS25R-bots]()
+- Enter a name for the bot to post with. (i.e. gcp-alert-service)
+- Click `Add bot integration`.
+- Wait until the UI displays the `API Token` and copy the string (i.e. xxxx-yyyyyyyyyyyy-zzzzzzzzzzzzzzzzzzzzzzzz)
+### 3) Configure the `Slack API Token`
+- Go back to the [Google Cloud Datastore Console](https://console.cloud.google.com/datastore) page.
+- Click `Create Entity`
+- Enter `Config` into the `Kind` textbox.
+- Add a property with a `Name` of '`name`' and `Value` of '`slackAPIToken`'. (Leave the `Type` as `String`)
+- Add another property with a `Name` of '`value`' and set the `Value` to the `Slack API Token` you generated in the previous step. (Leave the `Type` as `String`)
+- The result should look like this:
+![alt text](./Datastore1.jpg)
```

Comment:
Image didn't render in the [preview](https://github.com/spark2ignite/community/blob/462e6acc57bff5ca3f73d21107813feb50481fe4/tutorials/gSlack/index.md).